### PR TITLE
Fix more typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Continuation of  #12836

### CODE REVIEW

File: runtime/makemenu.vim [hunk 1]
```diff
@@ -327,7 +327,7 @@ SynMenu HIJK.Kivy:kivy
 SynMenu HIJK.KixTart:kix
 
 SynMenu L.Lace:lace
-SynMenu L.LamdaProlog:lprolog
+SynMenu L.LambdaProlog:lprolog
```
Comment (szepeviktor) [inline runtime/makemenu.vim:330]: This may be a bad fix.
(once more below)

File: runtime/syntax/ppd.vim [hunk 2]
```diff
@@ -15,7 +15,7 @@ syn match	ppdDefine	"\*[a-zA-Z0-9\-_]\+:"
 syn match	ppdUI		"\*[a-zA-Z]*\(Open\|Close\)UI"
 syn match	ppdUIGroup	"\*[a-zA-Z]*\(Open\|Close\)Group"
 syn match	ppdGUIText	"/.*:"
-syn match	ppdContraints	"^*UIConstraints:"
+syn match	ppdConstraints	"^*UIConstraints:"
```
Comment (szepeviktor) [inline runtime/syntax/ppd.vim:18]: This may be a typo in PPD.

**Comment by szepeviktor** [comment]
For the curious: `.typos.toml`

```toml
[files]
extend-exclude = [
    ".git/",
    "nsis/lang",
    "runtime/autoload/",
    "runtime/doc",
    "runtime/keymap/",
    "runtime/lang",
    "runtime/spell",
    "runtime/syntax",
    "runtime/tutor",
    "src/auto/configure",
    "src/configure.ac",
    "src/po/",
    "src/testdir/",
]
ignore-hidden = false

[default]
extend-ignore-re = [
    "0x[0-9a-z]{8}u",
    "2ND_BUTTON_PRESSED",
    "ahere is some text",
    '"a text t[^"]*"',
    "GROUPE BULL",
    "Christophe Devine",
    "@KnoP-01",
    "LANGuage",
    "Ned Konz",
    "Pyramis MIS-ES",
    "Mike Leary",
    "SungHyun Nam",
    "Synopsys",
    "Write-Ouput"
]
[default.extend-identifiers]
"CMD_enew" = "CMD_enew"
"HLF_DED" = "HLF_DED"
"KS_ND" = "KS_ND"
"ic_timout" = "ic_timout"
"one_adjust_nodel" = "one_adjust_nodel"
"O_WRONLY" = "O_WRONLY"
"t_nd" = "t_nd"
"T_ND" = "T_ND"
[default.extend-words]
"alloced" = "alloced"
"caf" = "caf"
"charcol" = "charcol"
"clen" = "clen"
"dout" = "dout"
"dum" = "dum"
"enew" = "enew"
"exten" = "exten"
"fo" = "fo"
"fpr" = "fpr"
"inex" = "inex"
"inout" = "inout"
"mak" = "mak"
"nore" = "nore"
"omlet" = "omlet"
"parm" = "parm"
"parms" = "parms"
"routeros" = "routeros"
"sav" = "sav"
"ser" = "ser"
"sover" = "sover"
"teh" = "teh"
"tolen" = "tolen"
"tpos" = "tpos"
"ue" = "ue"
"wee" = "wee"
"windo" = "windo"
```

File: runtime/syntax/ppd.vim (hunk 2 above)
Comment (szepeviktor) [reply runtime/syntax/ppd.vim:18]: No! This is a vim ID.
http://www.setec.org/~izaac/postscriptbooks/partners.adobe.com/asn/developer/pdfs/tn/5003.PPD_Spec_v4.3.pdf

**Comment by chrisbra** [comment]
thanks, I think it looks good. Let me include it

**Comment by chrisbra** [comment]
what have you been using for the spell-checking?

**Comment by szepeviktor** [comment]
> what have you been using for the spell-checking?

https://github.com/crate-ci/typos

**Comment by chrisbra** [comment]
Does it make sense to include that as part of the CI then?

**Comment by szepeviktor** [comment]
Yes!
https://github.com/szepeviktor/byte-level-care/blob/master/.github/workflows/spelling.yml#L31

But it needs some more work in addition to the above configuration.

**Pull request merged**